pkg/fingerprint/probes_test.go

```
@@ -0,0 +1,147 @@
+	}
+	if got := db.Probes[0].Matches[0].Product; got != "CustomSSH" {
+		t.Fatalf("product = %q, want CustomSSH", got)
+	}
+}
+
+func TestUpdateDefaultProbesValidatesAndWritesCache(t *testing.T) {
+	cachePath := filepath.Join(t.TempDir(), "naabu", probeFileName)
+	withProbeCachePath(t, cachePath)
+
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprint(w, tinyProbeDB)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)